Appraisals

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+appraise 'rails-4' do
+  gem 'activesupport', '~> 4.2'
```

Comment:
should these be `rails` instead of `activesupport` since the gem depends on all of `rails`?

Reply:
done